server/member/leader.go

```diff
@@ -248,10 +249,10 @@ func (m *Member) CampaignLeader(lease *LeaderLease, leaseTimeout int64) error {
 		Then(clientv3.OpPut(leaderKey, m.memberValue, clientv3.WithLease(lease.ID))).
 		Commit()
 	if err != nil {
-		return errors.WithStack(err)
+		return errs.ErrEtcdLeaderSave.FastGenByArgs()
```

Comment:
maybe return err directly, we need to know the actual error.

Reply:
Already fix it, PTAL @nolouch